site/layouts/pricing/list.html

```diff
@@ -58,10 +63,14 @@ <h3 class="">Team</h3>
                   <div class="pricing__fancy-price__dollar-sign pricing__fancy-price__dollar-sign--big">$</div>
                   <div class="pricing__fancy-price__price--big">30</div>
                 </div>
-                <div class="pricing-muted">per month</div>
+                <div class="pricing-muted text-medium">per month</div>
               </div>
             </div>
-            <div class="pricing__list">
+            <div class="pricing__btn__wrapper">
+              <a class="btn btn-success pricing__btn mb-3" href="https://app.checklyhq.com/signup?startOnTrial" rel="noopener">Try for free</a>
```

Comment:
We don't need the `startOnTrial` here anymore, right?

Reply:
No, there's a PR from Nico removing it